### PR TITLE
Include stars badge in README.md

### PR DESCRIPTION
**Pull request by shivarm**
# Preview

image

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
@gsmet @gastaldi

**Comment by gastaldi** [comment]
Cool, but I'd prefer having a single badge showing how many stars the repository has instead

**Comment by shivam-sharma7** [comment]
> Cool, but I'd prefer having a single badge showing how many stars the repository has instead.
> 
> Eg. GitHub Repo stars

@gastaldi should I add this in same place or on the Top where  all badges are present?

**Comment by gastaldi** [comment]
Probably next to the other badges, with the same Look'n Feel

**Comment by shivam-sharma7** [comment]
@gastaldi see

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?style=for-the-badge&logo=gitpod&logoColor=white)](https://gitpod.io/#https://github.com/quarkusio/quarkus/-/tree/main/)
 [![Supported JVM Versions](https://img.shields.io/badge/JVM-11--17--19-brightgreen.svg?style=for-the-badge&logo=openjdk)](https://github.com/quarkusio/quarkus/actions/runs/113853915/)
 [![Gradle Enterprise](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?style=for-the-badge&logo=gradle)](https://ge.quarkus.io/scans)
+[![GitHub Repo stars](https://img.shields.io/github/stars/quarkusio/quarkus?style=for-the-badge)](https://img.shields.io/github/stars/quarkusio/quarkus)
```
Comment (gastaldi) [inline README.md:12]:
```suggestion
[![GitHub Repo stars](https://img.shields.io/github/stars/quarkusio/quarkus?style=for-the-badge)](https://github.com/quarkusio/quarkus/stargazers)
```

**Review by gastaldi — CHANGES_REQUESTED** [review]
Pease squash your commits before we merge this

**Comment by gastaldi** [comment]
LGTM, but I think the commit and the PR title need to be reworded?

**Comment by shivam-sharma7** [comment]
> LGTM, but I think the commit and the PR title need to be reworded?

@gastaldi any suggetion for `commit` and `tittle`

**Comment by shivam-sharma7** [comment]
@gastaldi done!

**Review by gastaldi — APPROVED** [review]

**Pull request merged**